src/barfTurn.ts

```diff
@@ -227,12 +227,14 @@ const turns: AdventureAction[] = [
       get("_voteFreeFights") < 3,
     execute: () => {
       const isGhost = get("_voteMonster") === $monster`angry ghost`;
+      const isMutant = get("_voteMonster") === $monster`terrible mutant`;
 
       freeFightPrep(
         new Requirement([], {
           forceEquip: [
             $item`"I Voted!" sticker`,
             ...(!sober() && !isGhost ? $items`Drunkula's wineglass` : []),
+            ...(isMutant && !have($item`mutant crown`) ? $items`mutant arms, mutant legs`.filter((i) => have(i)) : []),
```

Comment:
```suggestion
            ...(isMutant && !have($item`mutant crown`) ? $items`mutant arm, mutant legs`.filter((i) => have(i)) : []),
```

If you have the eslint vscode plugin and `yarn install` (to install eslint-plugin-libram), it should catch these issues